src/components/loadable-item.cjsx

```diff
@@ -13,6 +13,7 @@ module.exports = React.createClass
   displayName: 'LoadableItem'
   propTypes:
     id: React.PropTypes.string.isRequired
+    options: React.PropTypes.object
     store: React.PropTypes.object.isRequired
```

Comment:
The `LoadableItem` component is extended with an `options` property.  This allows a user of the component to pass additional options to the store methods that are called by `LoadableItem`.  In the case of this PR, the store needs two id's for the endpoint, `courseId` and `roleId`.  The `roleId` is passed to the store in the options and from there to `api.coffee`